CONTRIBUTING.md

```diff
@@ -303,10 +303,32 @@ Various hints for developers
 - *DATALAD_TESTS_SSH*: 
   Skips SSH tests if this flag is **not** set
 - *DATALAD_LOGTRACEBACK*: 
-  Runs TraceBack function with collide set to True, if this flag is set to 'collide' 
+  Runs TraceBack function with collide set to True, if this flag is set to 'collide'.
   This replaces any common prefix between current traceback log and previous invocation with "..."
 - *DATALAD_TESTS_NOTEARDOWN*: 
   Does not execute teardown_package which cleans up temp files and directories created by tests if this flag is set
+- *DATALAD_SPECIAL_REMOTE*:
+  Specifies the name of the datalad [special remote](https://git-annex.branchable.com/special_remotes/).
+- *ARCHIVES_SPECIAL_REMOTE*:
+  Specifies the name of the datalad archive [special remote](https://git-annex.branchable.com/special_remotes/).
```

Comment:
checked -- indeed, just a constant in consts -- those aren't worth mixing into description of env variables since they aren't ones .  For consts, we just need to check how in sphinx to instruct to list those within the docs: now http://datalad.readthedocs.io/en/latest/generated/datalad.consts.html#module-datalad.consts  is quite boring